src/mlpack/prereqs.hpp

```diff
@@ -140,4 +140,7 @@ or upgrade Boost to 1.59 or newer.
 // We need to be able to mark functions deprecated.
 #include <mlpack/core/util/deprecated.hpp>
 
+// Include ready to use utility function for better workflow.
```

Comment:
```suggestion
// Include ready to use utility function to check sizes of datasets.
```
It might be nice to be a little more descriptive here. :)